app/utils.go

```diff
@@ -1,20 +1,36 @@
-//nolint
 package app
 
 import (
 	"fmt"
-	"io"
 	"io/ioutil"
 
-	"github.com/tendermint/tendermint/libs/log"
-	dbm "github.com/tendermint/tm-db"
-
-	"github.com/cosmos/cosmos-sdk/baseapp"
-	bam "github.com/cosmos/cosmos-sdk/baseapp"
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/staking"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
 )
 
+// SimulationOperations retrieves the simulation params from the provided file path
+// and returns all the modules weighted operations
+func SimulationOperations(app *GaiaApp, cdc *codec.Codec, config simulation.Config) []simulation.WeightedOperation {
```

Comment:
`simapp. SimulationOperations` receives a `SimApp` as parameter, so I can't use that util func atm. I'd need to extend `Simapp` to return the simulation manager (`app.sm`) and use a general `App` interface.

Reply:
I'd like to do that in a separate PR on the SDK tho

Reply:
will open an issue on SDK